content/pages/2022/pyladies_lunch.md

```diff
@@ -0,0 +1,32 @@
+---
+title: EuroSciPy 2022 - PyLadies Lunch
+url: 2022/pyladies_lunch.html
+save_as: 2022/pyladies_lunch.html
+section: euroscipy_2022
+template: page
+slug: pyladies_lunch_2022
+---
+
+# About PyLadies
+
+PyLadies is an international mentorship group with focus on helping
+under-represented developers become active participants and leaders in the
+Python open-source community. Our mission is to promote, educate and advance a
+diverse Python community through outreach, education, conferences, events and
+social gatherings.
+
+## PyLadies Lunch @ EuroSciPy
+
+![PyLadies Logo]({attach}../../static/2022/pylady_geek_wordmark_rainbow.png)
+
+Thanks to the supports of EuroSciPy organizers, our PyLadies Lunch will take
+place on Wednesday 31st of August.
+
+This is a great opportunity for women attending the conference to meet, make
+friends, and share experiences in a relaxing environment. So if you identify
+yourself as a woman, we would like to invite you to join us and make new
+friends or catch up with old friends.
+
+* **Who**: Anyone who identifies as a woman who is attending EuroSciPy 2022
```

Comment:
```suggestion
* **Who**: Anyone who identifies as a woman who is attending EuroSciPy 2022. If you don't identify as a woman but want to learn more about Pyladies you are also welcome to join.
```
Just a suggestion. Feel free to discard it.
my motive: that's how I learnt about Pyladies years ago: from a guy who attended Pyladies event on one of the conferences and told me about them

Reply:
I'm also happy with inviting allies. There are a lot of nice people who don't identify as female who want to learn about our work and how they can best support us and I think this is a good opportunity.